Reduce unveil needs by not implementing some 'ps' syscall.
Skip getAppMetrics() & getSystemMemoryInfo().

No network access, mute net errors during dns lookup (feature untested).

Run on Xvfb, getGPUInfo() return -1 (maybe UINT_MAX from vfb).

Index: electron/spec/disabled-tests.json
--- electron/spec/disabled-tests.json.orig
+++ electron/spec/disabled-tests.json
@@ -1,4 +1,10 @@
 [
+  "app module getAppMetrics() API returns memory and cpu stats of all running electron processes",
+  "app module configureHostResolver affects dns lookup behavior",
+  "app module getGPUInfo() API succeeds with basic GPUInfo",
+  "app module getGPUInfo() API succeeds with complete GPUInfo",
+  "process module renderer process process.getSystemMemoryInfo() returns system memory info object",
+  "process module main process process.getSystemMemoryInfo() returns system memory info object",
   "// NOTE: this file is used to disable tests in our test suite by their full title.",
   "BrowserWindow module BrowserWindow.loadURL(url) should emit did-fail-load event for files that do not exist",
   "Menu module Menu.setApplicationMenu unsets a menu with null",
